RefrigeratorExe/RefrigeratorExe/Program.cs

```diff
@@ -0,0 +1,297 @@
+﻿namespace RefrigeratorExe
+{
+    internal class Program
+    {
+       
+        static void Main(string[] args)
+        {
+
+            bool flag = true;
+            int numberPress;
+            List<Refrigerator> refrigerators = new List<Refrigerator>();
+            Refrigerator refrigerator = new Refrigerator("Samsong","Red",4);
+            refrigerators.Add(refrigerator);
+            BuildRefrigerators(refrigerators);
+            Console.WriteLine("\nWelcome To The Refrigerator App");
+            
+            while (flag) 
+            {
+                Console.WriteLine("");
+                Console.WriteLine("Press 1 to print out Item in Refrigerator");
```

Comment:
for advanced: 
have you learned about Dictionary? you can have mapping of number to its instruction

var instructions = new Dictionary<int, string>
{
    { 1, "print out Items in Refrigerator" },
    { 2, "print the amount of place left in Refrigerator" },
};

while (flag) 
{
    foreach (var instruction in instructions)
    {
        Console.WriteLine($"Press {instruction.Key}: {instruction.Value}");
    }